conf/app.item.supplyChainActors.index.routes

```diff
@@ -8,3 +8,9 @@ GET        /:itemIndex/supply-chain-actor/:actorIndex/eori-tin/:lrn
 POST       /:itemIndex/supply-chain-actor/:actorIndex/eori-tin/:lrn                        controllers.item.supplyChainActors.index.IdentificationNumberController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = NormalMode, itemIndex: Index, actorIndex: Index)
 GET        /:itemIndex/supply-chain-actor/:actorIndex/change-eori-tin/:lrn                 controllers.item.supplyChainActors.index.IdentificationNumberController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex:Index, actorIndex: Index)
 POST       /:itemIndex/supply-chain-actor/:actorIndex/change-eori-tin/:lrn                 controllers.item.supplyChainActors.index.IdentificationNumberController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex: Index, actorIndex: Index)
+
+
+GET        /:itemIndex/supply-chain-actor/:actorIndex/remove/:lrn                          controllers.item.supplyChainActors.index.RemoveSupplyChainActorController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = NormalMode, itemIndex: Index, actorIndex: Index)
+POST       /:itemIndex/supply-chain-actor/:actorIndex/remove/:lrn                          controllers.item.supplyChainActors.index.RemoveSupplyChainActorController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = NormalMode, itemIndex: Index, actorIndex: Index)
+GET        /:itemIndex/supply-chain-actor/:actorIndex/change-remove/:lrn                   controllers.item.supplyChainActors.index.RemoveSupplyChainActorController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex:Index, actorIndex: Index)
+POST       /:itemIndex/supply-chain-actor/:actorIndex/change-remove/:lrn                   controllers.item.supplyChainActors.index.RemoveSupplyChainActorController.onSubmit(lrn: LocalReferenceNumber, mode: Mode = CheckMode, itemIndex: Index, actorIndex: Index)
```

Comment:
Hello @BledarFeratiEY, I believe for this page we don't really need the mode, and we only need one GET and one POST route, just a little refactor. You might want to look at the remove Item PR to see how this was achieved previously.